index.js

```diff
@@ -89,5 +89,12 @@ export default {
         "selector-max-id": 0,
         "selector-pseudo-element-colon-notation": "double",
         "property-no-unknown": true,
+        "selector-class-pattern": [
+            "^([a-z][a-z0-9]*)(-[a-z0-9]+)*$",
```

Comment:
I'm not sure this is the correct pattern for how we name our classes – do you know how you'd check that?

Reply:
@beccadawson it's the pattern that's set in the default styelint config https://github.com/stylelint/stylelint-config-standard/blob/main/index.js#L95-L100 so I though it was the same pattern we already use?

Reply:
AFACIT we didn't actually use the standard config prior to v3 (https://github.com/AgePartnership/ap-stylelint-config/tree/v2.1.2), so think we'll actually have added a bunch of additional rules as part of the update to Stylelint 16 (and, sidenote, it looks like we're using the standard CSS config, not the SCSS config... I did a bad job at reviewing this 😅 )

So I think we just didn't have a check on this before!

Reply:
@beccadawson Oooh I did not realise that 😬😅 Shall I remove the use of the standard config (at least for now) then?

Reply:
Yeah maybe 😛 worth having on the radar to revisit our configs though (having a check that we're actually using bem conventions has been on my radar for a whlile!)